spec/support/disability_compensation_form/bdd_fe_submission.json

```diff
@@ -0,0 +1,95 @@
+{
+  "form526": {
+    "isVaEmployee": false,
+    "standardClaim": true,
```

Comment:
prematurely adding standardClaim, knowing that he FE will start sending it after https://github.com/department-of-veterans-affairs/va.gov-team/issues/11119 is complete

Reply:
I think it's fine to include 👍